.github/workflows/test.yml

```diff
@@ -53,5 +53,40 @@ jobs:
           name: test-report
           path: build/reports/tests/
       - run: ./gradlew buildPlugin
-      - run: ./gradlew runPluginVerifier -Pvalidation=lite
       - run: ./gradlew --stop
+      - uses: actions/upload-artifact@v4
+        with:
+          name: plugin.zip
+          path: './build/distributions/Sourcegraph-*.zip'
+  plugin-verifier:
```

Comment:
do they run in parallel?

Reply:
They cannot, plugin verifier depends on the uploaded artefacts.